app/uploaders/photo_uploader.rb

```diff
@@ -0,0 +1,64 @@
+class PhotoUploader < CarrierWave::Uploader::Base
+
+  include Cloudinary::CarrierWave
+
+  process eager: true # Force version generation at upload time.
+
+  process convert: 'jpg'
+
+  version :thumnail do
+    resize_to_fit 256, 256
+  end
+
+  version :bright_face do
+    cloudinary_transformation effect: "brightness:30", radius: 20,
+      width: 150, height: 150, crop: :thumb, gravity: :face
+  end
+
+end
+
```

Comment:
We can delete everything below this line i believe

Reply:
keeping just as a reference if we do more things later with cloudinary